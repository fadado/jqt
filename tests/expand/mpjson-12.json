// Macros
&define{T}{Mister }&
{
    "firstName": "&T&_{}John F.",
    "lastName": "Kennedy",
    "age": 25,
    "gender": { "type": "male" }
}
// vim:sw=4:ts=4:ai:et:syntax=javascript
