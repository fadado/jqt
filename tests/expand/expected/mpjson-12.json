{
    "firstName": "Mister John F.",
    "lastName": "Kennedy",
    "age": 25,
    "gender": { "type": "male" }
}

